website/docs/guides/container_service_extension_4_0_cluster_management.html.markdown

```diff
@@ -0,0 +1,271 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Container Service Extension v4.0 Kubernetes clusters management"
+sidebar_current: "docs-vcd-guides-cse-4-0-cluster-management"
+description: |-
+  Provides guidance on provisioning Kubernetes clusters using Container Service Extension v4.0
+---
+
+# Container Service Extension v4.0 Kubernetes clusters management
+
+## About
+
+This guide explains how to create, update and delete TKGm clusters in a VCD appliance with Container Service Extension v4.0
+installed, using Terraform.
+
+We will use the [`vcd_rde`][rde] resource for this purpose.
+
+~> This section assumes that your CSE installation was done following the [CSE v4.0 installation guide][cse_install_guide].
+That is, CSE Server should be up and running and all elements must be working.
+
+## Pre-requisites
+
+-> Please read also the pre-requisites section in the [CSE documentation][cse_docs].
+
+In order to complete the steps described in this guide, please be aware:
+
+* CSE v4.0 must be installed. Read the [installation guide][cse_install_guide] for more information.
+* Terraform provider needs to be v3.9.0 or above.
+* CSE Server must be up and running.
+
+## Creating a Kubernetes cluster
+
+-> You can have a look at a working example of a TKGm cluster [here][cluster]. It is encouraged to read the following
+section to understand how it works.
+
+To be able to create a TKGm cluster, one needs to prepare a [`vcd_rde`][rde] resource. In the [proposed example][cluster],
+this RDE is named `k8s_cluster_instance`. The important arguments to take into account are:
+
+- **`resolve` must be always `false`**, because the CSE Server is responsible for performing the RDE resolution when the
+  TKGm cluster is completely provisioned, so Terraform should not interfere with this process.
+- **`resolve_on_removal` must be always `true`**, because the RDE is resolved by the CSE Server and not by Terraform. If one
+  wants to execute a Terraform destroy without the RDE being resolved, the operation will fail. Being `true` assures that Terraform
+  can perform a Terraform destroy in every case. See ["Deleting a Kubernetes cluster"](#deleting-a-kubernetes-cluster) section for more info.
+
+The [`vcd_rde`][rde] argument `input_entity` is taking the output of the Terraform built-in function `templatefile`, that references
+a JSON template that you can find [here][tkgmcluster_template]. This function will set the correct values to the following
```

Comment:
```suggestion
a JSON template that is located at [here][tkgmcluster_template]. This function will set the correct values to the following
```
We should not mix the direct "you" with the impersonal "one" in the document

Reply:
Fixed!